src/components/RecipeDetails.tsx

```diff
@@ -56,6 +65,11 @@ const RecipeDetailsCard = styled.div`
   position: relative;
   padding-left: 70px;
   overflow: hidden;
+  opacity: 0;
+  animation-name: ${recipeDetailsTransition};
+  animation-duration: 2.5s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: 1;
```

Comment:
I believe you can shorthand this to something like "animation: ${textFlash} .8s linear infinite alternate;"